### PR TITLE
JBPM-5617 error for forms with a Slider component based on an Integer field

### PR DESCRIPTION
**Pull request by alessioscalici**
Fixing the wrong behavior providing an appropriate converter.

@pefernan @jsoltes could you take a look?

### CODE REVIEW

File: kie-wb-common-forms/kie-wb-common-dynamic-forms/kie-wb-common-dynamic-forms-client/src/test/java/org/kie/workbench/common/forms/dynamic/client/rendering/renderers/SliderFieldRendererTest.java [hunk 1]
```diff
@@ -0,0 +1,100 @@
+package org.kie.workbench.common.forms.dynamic.client.rendering.renderers;
```
Comment (manstis) [inline kie-wb-common-forms/kie-wb-common-dynamic-forms/kie-wb-common-dynamic-forms-client/src/test/java/org/kie/workbench/common/forms/dynamic/client/rendering/renderers/SliderFieldRendererTest.java]: License

File: kie-wb-common-forms/kie-wb-common-forms-commons/kie-wb-common-forms-common-rendering/kie-wb-common-forms-common-rendering-client/pom.xml [hunk 2]
```diff
@@ -68,6 +68,11 @@
       <artifactId>gwtmockito</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
```
Comment (manstis) [inline kie-wb-common-forms/kie-wb-common-forms-commons/kie-wb-common-forms-common-rendering/kie-wb-common-forms-common-rendering-client/pom.xml]: Wozza. Really? Uberfire "table" widget for a Forms test?

Comment (alessioscalici) [inline kie-wb-common-forms/kie-wb-common-forms-commons/kie-wb-common-forms-common-rendering/kie-wb-common-forms-common-rendering-client/pom.xml]: sorry, my oversight. fixed

File: kie-wb-common-forms/kie-wb-common-dynamic-forms/kie-wb-common-dynamic-forms-client/src/test/java/org/kie/workbench/common/forms/dynamic/client/rendering/renderers/SliderFieldRendererTest.java (hunk 1 above)
Comment (alessioscalici) [inline kie-wb-common-forms/kie-wb-common-dynamic-forms/kie-wb-common-dynamic-forms-client/src/test/java/org/kie/workbench/common/forms/dynamic/client/rendering/renderers/SliderFieldRendererTest.java]: Thanks, fixed

**Comment by jsoltes** [comment]
@manstis It looks like this got merged before I managed to have a look... There is actually a regression connected probably to this pull-request. One thing is, that it is not possible to add unbinded slider to the canvas. With binded slider, there is an error everytime user opens form with slider or tries to add a slider field to the form: `Unable to complete your request. The following exception occurred: Exception caught: Exception caught: (TypeError) : Cannot read property 'insertBefore' of null..` See the attached video.
[reproducer.zip](https://github.com/kiegroup/kie-wb-common/files/1148330/reproducer.zip)

**Comment by manstis** [comment]
@jsoltes Please see https://github.com/errai/errai/pull/268

**Comment by jsoltes** [comment]
@manstis thanks

**Pull request merged**